scripts/build_kb.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+if [[ -z ${BINARY_PATH+1} ]];
+then
+  source "$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)"/set_vars.sh
+fi
+
+"${BINARY_PATH}/sc-builder" -f --clear -c "${CONFIG_PATH}" -i "${APP_ROOT_PATH}/repo$1.path" -o "${APP_ROOT_PATH}"/kb.bin
```

Comment:
I don't think this is flexible enough. `-i` should probably have a default value, but providing an additional argument should fully override it. In my opinion something akin to [this](https://github.com/ostis-ai/sc-machine/blob/4026a47b1aaa6fea3febae1363629761208f6c5a/scripts/docker_entrypoint.sh#L82-L100) should do if what you want to achieve is script's independence on implementation.